### ISSUE TITLE
Heading hierarchy in the sidebar 

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#2024]
Noticed while checking #2020. See also #503.

The whole heading hierarchy in Gutenberg is still to address, there should be a main h1 to indicate the page purpose, then the main sections should have h2 headings, and so on. Heading levels should not be skipped.

In the sidebar, currently the settings sections use an h3, They should probably be changed, depending also on the other headings to add in the page.

Current hierarchy:

![screen shot 2017-07-26 at 14 48 04](https://user-images.githubusercontent.com/1682452/28627565-7c587272-7222-11e7-8818-0f0554207474.png)

Many of the headings in the screenshot above are actually in the demo post content. The "All that you can embed!" one is the last heading block in the content.

"Gallery Settings" and "Block Settings" are actually in the sidebar, but given this structure they're identified as children of "All that you can embed!" 🙁

**Comment by afercia** [comment]
See also #503
